.travis.yml

```diff
@@ -12,9 +12,16 @@ env:
 before_install:
   # See https://github.com/travis-ci/travis-ci/issues/4629#issuecomment-239493916
   - rm ~/.m2/settings.xml
-  - curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
+  - if [ ! -f ~/.jabba/jabba.sh ]; then curl -L -v --retry 5 -o jabba-install.sh https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh && bash jabba-install.sh; fi
+  - . ~/.jabba/jabba.sh
+
+install:
+  # Using ~ makes jabba installs the latest version for that major JDK release
+  - jabba install $(jabba ls-remote "adopt@~1.$TRAVIS_JDK.0-0" --latest=patch)
+  - unset _JAVA_OPTIONS
+  # Using ~ with ls works in a similar way to jabba install described above
+  - export JAVA_HOME="$JABBA_HOME/jdk/$(jabba ls "adopt@~1.${TRAVIS_JDK}.0-0" | head -1)" && export PATH="$JAVA_HOME/bin:$PATH" && jabba use "$_" && java -Xmx32m -version
```

Comment:
this is what we have in lagom-java.g8

Build is failing because JAVA_HOME is not set. No idea why it's complaining here (and now) and not in other projects.

Reply:
Apparently it _has been_ failing for a while now (https://travis-ci.com/github/lagom/lagom-samples/jobs/355159681#L230).

Good to see it fixed once and for all. 

PS. the JDK8 jobs for gRPC are broken because this build uses the latest JDK patch of JDK8 (252 as of this writing). It's a known bug so I'll merge.